sdk/nodejs/runtime/settings.ts

```diff
@@ -49,6 +49,11 @@ export interface Options {
      */
     readonly project?: string;
 
+    /**
+     * The root directory of the current project. This is the location of the Pulumi.yaml file.
+     */
+    readonly projectRoot?: string;
```

Comment:
RootDirectory, this matches what we send to providers. See the Handshake method.